src/sql/azdata.d.ts

```diff
@@ -3387,6 +3387,7 @@ declare module 'azdata' {
 
 		/**
 		 * SplitView height
+		 * @deprecated use splitViewSize instead
 		 */
 		splitViewHeight: number | string;
```

Comment:
You should still fall back to this value if splitViewSize isn't defined

Reply:
You should also make this optional - otherwise anyone trying to use this type is going to be told they need to define a splitViewHeight value as well.

Reply:
if `splitViewSize` isn't defined, then the dimensions of the splitViewContainer are used. Should I make it use the old `splitViewHeight` if that's set, then only use `splitViewSize` if it isn't?